pkgs/test/lib/src/runner/browser/browser_manager.dart

```diff
@@ -238,8 +236,17 @@ class BrowserManager {
       });
 
       try {
-        controller = deserializeSuite(path, currentPlatform(_runtime),
-            suiteConfig, await _environment, suiteChannel, message);
+        controller = deserializeSuite(
+            path,
+            currentPlatform(_runtime),
+            suiteConfig,
+            await _environment,
+            suiteChannel,
+            message, gatherCoverage: () async {
+          var browser = _browser;
+          if (browser is Chrome) return browser.gatherCoverage();
+          return {};
```

Comment:
should we warn here or anything?

Reply:
This will produce an empty result which is consistent with the current behavior. I might follow up with a PR that warns at the time of arg parsing.